### ISSUE TITLE
Screamachine flipped HP bars not showing

### ISSUE THREAD

**Issue by wac** [SotMSteamMods/CauldronMods#1248]
**Describe the bug**
The flipped versions of Screamachine band members don't show HP bars.  When you inspect the card the current HP is printed on them though.

**Screenshots**
![image](https://user-images.githubusercontent.com/28958/115324160-0303c880-a14f-11eb-9e5c-b33dc8037349.png)

**Comment by LNBertolini** [comment]
Yeah, this is a hardcoded part of the UI. Flipped villain character cards that aren't the "main" villain character card lose their HP bar unless they're the Talisman.
